### PR TITLE
New actions

### PR DESCRIPTION
**Pull request by kingsleydon**
New actions

`astar_bridge_to_phala`
`phala_bridge_to_polkadot`
`phala_native_transactor`
`khala_native_transactor`
`polkadot_bridge_to_phala`
`polkadot_bridge_to_moonbeam`
`polkadot_bridge_to_astar`
`polkadot_bridge_to_astar_evm`
`polkadot_native_transactor`

### CODE REVIEW

File: contracts/index_executor/src/actions/polkadot/mod.rs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                MOONBEAM_PARACHAIN_ID,
+                AccountType::Account20,
+            )),
+        ),
+        (
+            String::from("polkadot_bridge_to_astar_evm"),
+            Box::new(xcm_v3::PolkadotXcm::new(
+                ASTAR_PARACHAIN_ID,
+                AccountType::Account32,
+                true,
```
Comment (tolak) [inline contracts/index_executor/src/actions/polkadot/mod.rs:32]: And does this mean Astar only support XCM V3 message to be send directly to their EVM?

**Review by tolak — APPROVED** [review]
LGTM, had some suggestions and questions

File: contracts/index_executor/src/actions/polkadot/mod.rs (hunk 1 above)
Comment (kingsleydon) [reply contracts/index_executor/src/actions/polkadot/mod.rs:32]: Similar to moonbeam, the parameters of this transaction were obtained from the astar official app, so I cannot be absolutely certain.

File: contracts/index_executor/src/actions/polkadot/mod.rs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            String::from("polkadot_bridge_to_phala"),
+            Box::new(xcm_v3::PolkadotXcm::new(
+                PHALA_PARACHAIN_ID,
+                AccountType::Account32,
+                false,
+            )),
+        ),
+        (
+            String::from("polkadot_bridge_to_moonbeam"),
+            Box::new(xcm_v2::PolkadotXcm::new(
```
[the remark this answers is not in the document]
Comment (kingsleydon) [reply contracts/index_executor/src/actions/polkadot/mod.rs:22]: This transaction was obtained from the official application of moonbeam, due to the lack of relevant documentation, the preference here is to ensure it to be entirely identical to the extrinsic constructed by the official App.

**Pull request merged**